substra/sdk/client.py

```diff
@@ -52,40 +52,70 @@ def wrapper(*args, **kwargs):
 
 
 class Client(object):
-    """Create the client.
+    """Create a client
 
-    Set the `backend` to 'local' to use the local debugging.
-    One client corresponds to one profile.
+    Args:
+
+        url (typing.Optional[str], optional): URL of the Substra platform. Mandatory
+            to connect to a Substra platform. If no URL is given and debug is True, all
+            assets must be created locally.
+            Defaults to None.
+
+        token (typing.Optional[str], optional): Token to authenticate to the Substra platform.
+            If no token is given, use the 'login' function to authenticate.
+            Defaults to None.
+
+        retry_timeout (int, optional): Number of seconds before attempting a retry call in case
+            of timeout.
+            Defaults to 5 minutes.
+
+        version (str, optional): API version, do not change it. Defaults to '0.0'.
+
+        insecure (bool, optional): If True, the client can call a not-certifed backend. This is
+            for debugging purposes.
+            Defaults to False.
+
+        debug (bool, optional): Whether to use the default or debug mode.
+            In debug mode, new assets are created locally but can access assets from
+            the deployed Substra platform. The platform is in read-only mode.
+            Defaults to False.
     """
 
     def __init__(
         self,
         url: typing.Optional[str] = None,
```

Comment:
I think it is a shame to let go of the fully local backend when it is so simple to keep it as is, but I am not sure how useful it will be (and I am definitely partial on this !), maybe ask the users ?

Reply:
Having an optional url seems good to me. 
Maybe we could make the docstring clearer:
 
> If no URL is given and debug is True, all assets must be created locally.

to 

> If no URL is given and debug must be defined as True, all assets will be created locally.